tensorflow/docs_src/programmers_guide/custom_estimators.md

```diff
@@ -362,7 +362,7 @@ model's loss. This is the
 that will be optimized.
 
 We can calculate the loss by calling @{tf.losses.sparse_softmax_cross_entropy}.
-The value returned by this function will be lowest, approximately 0,
+When the value returned by this function will be lowest, approximately 0,
```

Comment:
How about this:

The value returned by this function will be approximately 0 at lowest, when the probability of the correct class (at index `label`') is near 1.0.

Reply:
That sounds better. Thanks. 
Should I update the PR with it ?

Reply:
I just patched it.